### ISSUE TITLE
Decreased acetylcholine level 

### ISSUE THREAD

**Issue by sarahstofel** [obophenotype/human-phenotype-ontology#7067]
**Preferred term label:**

Decreased acetylcholine level

**Synonyms**

Decreased Ach level

**Definition (free text, please give PubMed ID)**

A reduction in the normal amount of acetylcholine.
PMID:12163621

**Parent term (use hpo.jax.org/app)**

abnormal synaptic transmission

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

OMIM: 104300

**Your nano-attribution (ORCID)**

https://orcid.org/0000-0003-2307-1226

**Comment by pnrobinson** [comment]
@sarahstofel  -- the PMID (https://pubmed.ncbi.nlm.nih.gov/12163621/) describes regional cerebral blood flow decreases and not decreases in acetylcholine.
The scope of the HPO is for abnormalities that are measured as a part of the clinical encounter in some way. It would not be appropriate to include a term that represents an inference that there must be reduced ACh because of effects of AChEI therapy. I am not an expert here, can you please comment?

**Comment by pnrobinson** [comment]
I think that for now this is not a measurement made in standard clinical practice and thus is not in scope for HPO. Please reopen if you disagree.
